src/modules/print/SettingsPrintView.js

```diff
@@ -73,12 +73,15 @@ class SettingsPrintView extends PrintView {
 
   renderDeviceMeta() {
     const device = deviceName(this.manufacturer) || t('Unknown');
+    const { serial } = this.deviceMeta;
+    const serialText = serial === 'Unknown' ? '' : ` › ${t('Serial Number: {{serial}}', { serial })}`;
```

Comment:
Hiding the "Serial Number: Unknown" text here to be consistent with web, where it is not displayed.